vendor/assets/stylesheets/bootstrap_toggle.css

```diff
@@ -25,4 +25,4 @@
 .toggle-off.btn-sm{padding-left:20px}
 .toggle.btn-xs{min-width:35px;min-height:22px}
 .toggle-on.btn-xs{padding-right:12px}
-.toggle-off.btn-xs{padding-left:12px}
```

Comment:
@colinxfleming I'm not sure why the ci/circleci isn't accepting the move. I removed the extra lines. Do you see anything else wrong? Github says the move would merge just fine.